workflows/dragen-somatic-pipeline/3.9.3/dragen-somatic-pipeline__3.9.3.cwl

```diff
@@ -56,11 +56,24 @@ requirements:
   SchemaDefRequirement:
     types:
       - $import: ../../../schemas/fastq-list-row/1.0.0/fastq-list-row__1.0.0.yaml
-      - $import: ../../../schemas/predefined-mount-path/1.0.0/predefined-mount-path__1.0.0.yaml
 
 # Declare inputs
 inputs:
   # File inputs
+  # Option 1
+  fastq_list:
+    label: fastq list
+    doc: |
+      CSV file that contains a list of FASTQ files for normal sample
```

Comment:
Maybe worth stating that for both `fastq_list` and `tumor_fastq_list` that the `read_1` and `read_2` components must be presigned urls.

Reply:
Good point.